scripts/configure-dh.sh

```diff
@@ -131,6 +133,28 @@ fi
 
 echo "OK"
 
+# Reads Model URL for lightspeed plugin
+# Optional: If no URL is entered, lightspeed plugin will not be configured
+if [ -z "${LIGHTSPEED_MODEL_URL}" ]; then
+    read -p "Enter your model URL for lightspeed (Optional): " LIGHTSPEED_MODEL_URL
+
+    # Reads API token for lightspeed plugin
+    # Optional: If no token is entered, lightspeed plugin will not use authenticated communication
+    if [ -z "${LIGHTSPEED_API_TOKEN}" ]; then
+        read -p "Enter API token for lightspeed (Optional): " LIGHTSPEED_API_TOKEN
+    fi
+    echo "OK"
+fi
+if [ ! -z "${LIGHTSPEED_MODEL_URL}" ]; then
+    # make sure the target url ends in /v1
+    if ! [[ $LIGHTSPEED_MODEL_URL =~ \/v1$ ]]; then
+        LIGHTSPEED_MODEL_URL="$LIGHTSPEED_MODEL_URL/v1"
+    fi
+
+    plugin_sha=$(npm view @janus-idp/backstage-plugin-lightspeed dist.integrity)
+    yq -i ".plugins.[0].integrity = \"${plugin_sha}\"" $BASE_DIR/optional-plugins/lightspeed-plugins.yaml
+fi
```

Comment:
Assuming the prompts would exist in both `configure.sh` and `configure-dh.sh`, any unfilled optional variable from the first prompt would fall through to the second script as unset again.

Some other value would be needed to discriminate between an optional value that wasn't processed, and one that was left empty on purpose. Otherwise we'll get duplicate prompts when calling `configure.sh`. Got any in mind?

Reply:
Anyway, here is the enablement variable and prompt in `configure.sh`

Reply:
> any unfilled optional variable from the first prompt would fall through to the second script as unset again

@jrichter1 This is a good point, the optional input would prompt twice if left blank the first time. To counter this, it might make sense in each of the scripts under `/scripts` to include an additional `BYPASS_OPTIONAL_INPUT` flag variable to include in the checks for the optional prompts alongside the set conditions.

`configure.sh` can have this set going into each of these scripts to bypass the same optional input prompts regardless if they are set.

Reply:
I've basically listed all the processed optional variables from `config.sh` into `BYPASS_OPTIONAL_INPUT`, so if any of them is found in there, its second prompt will be skipped. Can you give it a try?

Reply:
I was able to get everything up and running with the new additions, my lgtm still stands @jrichter1 @michael-valdron